packages/core/pluggableElementTypes/WidgetType.ts

```diff
@@ -10,10 +10,8 @@ export default class WidgetType extends PluggableElementBase {
 
   HeadingComponent?: ComponentType<{ model: IAnyStateTreeNode }>
 
-  ReactComponent: ComponentType<{
-    model: IAnyStateTreeNode
-    session: IAnyStateTreeNode
-  }>
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ReactComponent: React.FC<any>
```

Comment:
Note that errors like this are because of

```
plugins/data-management/src/index.ts:67:9 - error TS2322: Type 'typeof AddTrackWidget' is not assignable to type 'ComponentType<{ model: IAnyStateTreeNode; session: IAnyStateTreeNode; }>'.
  Type 'typeof AddTrackWidget' is not assignable to type 'FunctionComponent<{ model: IAnyStateTreeNode; session: IAnyStateTreeNode; }>'.
    Types of parameters '__0' and 'props' are incompatible.
      Type 'PropsWithChildren<{ model: IAnyStateTreeNode; session: IAnyStateTreeNode; }>' is not assignable to type '{ model: ModelInstanceTypeProps<{ id: IOptionalIType<ISimpleType<string>, [undefined]>; type: ISimpleType<"AddTrackWidget">; view: IMaybe<IReferenceType<IAnyType>>; }> & { ...; } & { ...; } & { ...; } & IStateTreeNode<...>; }'.
        Types of property 'model' are incompatible.
          Type 'IAnyStateTreeNode' is not assignable to type 'ModelInstanceTypeProps<{ id: IOptionalIType<ISimpleType<string>, [undefined]>; type: ISimpleType<"AddTrackWidget">; view: IMaybe<IReferenceType<IAnyType>>; }> & { ...; } & { ...; } & { ...; } & IStateTreeNode<...>'.
            Type 'IAnyStateTreeNode' is missing the following properties from type 'ModelInstanceTypeProps<{ id: IOptionalIType<ISimpleType<string>, [undefined]>; type: ISimpleType<"AddTrackWidget">; view: IMaybe<IReferenceType<IAnyType>>; }>': id, type, view

67         ReactComponent: AddTrackReactComponent,
           ~~~~~~~~~~~~~~

```


Basically IAnyStateTreeNode is underspecified to properly typescript model/session